README.md

```diff
@@ -2,34 +2,35 @@
 
 # NuoDB Collector
 
-[![Build Status](https://travis-ci.org/nuodb/nuodb-collector.svg?branch=master)](https://travis-ci.org/nuodb/nuodb-collector)
+[![nuodb](https://circleci.com/gh/nuodb/nuodb-collector.svg?style=svg)](https://circleci.com/gh/nuodb/nuodb-collector)
 
 # Introduction
 
 Most modern application monitoring systems consist of the following 3
 core components:
 
-* Collector             — daemon(s) to gather metrics such as this repository
-* Time Series database  — for storage of real-time, high volume metrics (e.g. InfluxDB, Prometheus, LogStash)
-* Query & Visualization — that enables visual monitoring and root cause analysis (e.g. Grafana)
+- Collector — daemon(s) to gather metrics such as this repository
+- Time Series database — for storage of real-time, high volume metrics (e.g. InfluxDB, Prometheus, LogStash)
+- Query & Visualization — that enables visual monitoring and root cause analysis (e.g. Grafana)
 
 NuoDB Collector utilizes a popular open-source collector - `telegraf`.
 It's designed to be used alongside a NuoDB engine process to collect metrics from the engine and publish those metrics to a time series database.
 
 Built into this container are 4 input plugins to collect metrics from the NuoDB engine:
 
-1.  `metrics` - collects the [Engine Metrics](https://doc.nuodb.com/nuodb/4.0.x/reference-information/metrics-published-by-database-processes/)  on a
+1.  `metrics` - collects the [Engine Metrics](https://doc.nuodb.com/nuodb/4.0.x/reference-information/metrics-published-by-database-processes/) on a
```

Comment:
This is a dead link. Can you change it to https://doc.nuodb.com/nuodb/latest/reference-information/metrics-published-by-database-processes ?